bundle/stable/eclipse-che/manifests/che-operator.clusterserviceversion.yaml

```
@@ -917,15 +916,15 @@ spec:
                       - name: OPERATOR_NAME
                         value: che-operator
                       - name: CHE_VERSION
-                        value: 7.66.0
+                        value: 7.67.0
                       - name: RELATED_IMAGE_che_server
-                        value: quay.io/eclipse/che-server@sha256:a9c40f83a81af095c0aa3a6f57c202f15f87e579dc317ee6750bd0e7bdfe88c2
+                        value: quay.io/eclipse/che-server@sha256:9469cb76435e023a402765349601d9b879abc6425e43f0b49ae2a337c780bd0d
                       - name: RELATED_IMAGE_dashboard
-                        value: quay.io/eclipse/che-dashboard@sha256:4e3e085fb402e84d3a13bb5356bfe3b8f26f4b5af11428e0213326d927fe0d3e
+                        value: quay.io/eclipse/che-dashboard@sha256:5805e8e13845e162149285ee14b7b28af9c8b8098d1e6219c426ea848905cc6c
                       - name: RELATED_IMAGE_plugin_registry
-                        value: quay.io/eclipse/che-plugin-registry@sha256:563cb2180b02d7d8e8a9eaa66532a598fbab4ba3ad93925d924d13e3e0f6718d
+                        value: quay.io/eclipse/che-plugin-registry@sha256:79fbcb85ff84a5786d1fdea5684ab3e0662a7b21f4cb3cd13d2c61b653931b70
```

This image digest does not appear to exist. I'm not sure where this PR got this SHA, but I suspect it has to do with the failed [`release-plugin-registry`](https://github.com/eclipse-che/che-plugin-registry/actions/runs/5005985666/jobs/8972814854) job recently. 

The 7.67.0 plugin registry (that was pushed by a secondary run of the failed job) appears to be `fa87bcad1edd184f7cb5527d75aec1e73e42f66195e458562dc6ee0eb8877326`


```suggestion
                        value: quay.io/eclipse/che-plugin-registry@sha256:fa87bcad1edd184f7cb5527d75aec1e73e42f66195e458562dc6ee0eb8877326
```

Someone who is more familiar with this piece of the release process should confirm though (where did the bad SHA come from in the first place?)